.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,44 @@
+#
+# This agent will automatically record (some) CTO.ai Delivery & LifeCycle
+# Metrics for use with the CTO.ai Delivery Metrics dashboard.
+#
+# Installation:
+#
+#   1. GitHub -> Settings -> Secrets -> New Secret
+#     - Create CTOAI_TEAM_ID secret using your CTO.ai-issued Team Id.
+#     - Create CTOAI_EVENTS_API_TOKEN secret using your CTO.ai-issued API Token.
+#   2. Drop this file into your GitHub repository at your-repo/.github/workflows/
+#
+
+on:
+  pull_request: 
+    types: [closed]
```

Comment:
successful merges also come in as `closed`, so you will get a deployment fail before the deployment success on push in the next workflow.

Reply:
Ok, we'll need to look into adding an `if` condition in the example. I didn't get that far last night.